src/app.js

```diff
@@ -1,9 +1,17 @@
 const express = require('express');
-
+const mongoose = require('mongoose');
+const user = require('./models/users')
 const app = express()
 
-app.post('/signup', (req, res) => {
-    res.send('ok')
+app.use(express.urlencoded({ extended: false }));
+app.use(express.json())
+
+mongoose.connect(process.env.MONGO_URI).catch(error=>console.log(error.message))
+
+app.post('/signup', async(req, res) => {
```

Comment:
You may need to follow a convention when naming your URLs
Here is a good resource to consider
https://restfulapi.net/resource-naming/

Also, it can be beneficial to use  a concept known as API versioning
You can ignore this for now, but here is a resource that you can take  a look at https://voskan.host/documentation/index.php/docs/nodejs/building-restful-apis-with-express-js/api-versioning/